hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHAWithData.java

```diff
@@ -69,6 +70,38 @@ public void testAllOMNodesRunning() throws Exception {
     createKeyTest(true);
   }
 
+  @Test
+  public void testBucketLinkOps() throws Exception {
```

Comment:
We already have acceptance test for links, but currently it is only run in non-HA setup.  Instead of adding integration test, I think we should run the existing test in HA setup, too:

```
execute_robot_test scm basic/links.robot
```

could be added to `hadoop-ozone/dist/src/main/compose/ozonesecure-om-ha/test.sh` and maybe `hadoop-ozone/dist/src/main/compose/ozone-om-ha-s3/test.sh`.

This confirms that the bug and the fix.

However, I also see two of the test cases failing with the fix:

```
Cannot follow link without read access                                | FAIL |
ACL verified on source bucket                                         | FAIL |
```

Reply:
Thanks for the tip. Enabled docker-compose tests.